api/backend/blueflood/blueflood.go

```diff
@@ -132,20 +140,61 @@ func (b *blueflood) FetchSingleSeries(request api.FetchSeriesRequest) (api.Times
 	}
 	queryResolution := b.config.bluefloodResolution(
 		request.Timerange.Resolution(),
-		request.Timerange.Start())
+		request.Timerange.Start(),
+	) // The query resolution will be downsampled automatically based on the timerange?
 
+	// <Sample the data>
 	queryUrl, err := b.constructURL(request, sampler, queryResolution)
 	if err != nil {
 		return api.Timeseries{}, err
 	}
-
-	// Issue GET to fetch metrics
 	parsedResult, err := b.fetch(request, queryUrl)
 	if err != nil {
 		return api.Timeseries{}, err
 	}
+	// </Sample the data>
+
+	// <Sample the full resolution data>
+	fullResolutionRequest := request
+
+	// The "now" as an epoch in milliseconds
+	now := getNow().Unix() * 1000
```

Comment:
The goal of clipping the data to be recent-ish was to reduce the amount of unneeded full-resolution points being requested. 

It's probably simpler to just request the same full timerange for full resolution data.

Will there be any performance impact from this? Or will this not be a problem?